admin/src/OptionsSetter.js

```diff
@@ -150,6 +150,7 @@ class OptionsSetter extends React.Component {
             </th>
```

Comment:
@robmadole something I'm concerned about is the confusion there may be about what actually happens when clicking that box to "Use Pro". What this plugin actually does is to set the CDN url to `pro.` instead of `use.`.

So if someone were to just try it without subscribing to Pro _and_ setting up referrer domains, it will just appear to break all of their icons. And it may not be apparent that it was checking "Use Pro" that caused the break. Because this is not a normal client requirement that registered a conflict, there will be no other visual indication of the problem. This might be the _only_ issue that could cause such a silent catastrophe (other than nasty unregistered clients that we're not able to detect and which break everything).

So it seems like we might want something more here to keep people from triggering this foot gun.

But if you think we can live without putting a safety on the foot gun for this release, I'll go along.

Reply:
Even if it said something like "make sure you've subscribed and enabled your CDN domains before checking this box", that might at least get them to read the linked page more thoroughly before pulling the trigger.

Reply:
Yeah, it's certainly not ideal. We could add a link to "Configure your domains". That would hint to them that if things aren't working there may be additional steps that they have missed.

Reply:
Would that be _instead_ of the current link? Or in addition (like below the checkbox, while keeping the existing link out to the right)?